bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt

```diff
@@ -91,6 +96,11 @@ abstract class ImportIntoModelServer @Inject constructor(of: ObjectFactory) : De
                     logger.info("Got root node: {}", rootNode)
                     logger.info("Calculating diff...")
                     ModelImporter(rootNode, continueOnError.get()).importFilesAsRootChildren(files)
+
+                    logger.info("Setting meta properties...")
+                    for ((key, value) in metaProperties.get()) {
+                        rootNode.setPropertyValue(IProperty.fromName(key), value)
+                    }
```

Comment:
Are there any reserved properties already on the root node the plugin implementation should shield against from blindly accepting them?

Reply:
As far as I know, the root node doesn't contain any properties by default.